### PR TITLE
Patch v19.4.0: Pull stellar-core 19.4.0 inti kinesis-core

### PR DESCRIPTION
**Pull request by ShailjaMaheshwari**

### CODE REVIEW

File: KINESIS.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Development Environment Setup
+
+- [Download & install vscode](https://code.visualstudio.com/download)
+- Open the kinesis-core project in vscode and click on "Reopen in Container" when asked
+![image](https://user-images.githubusercontent.com/29750/203445568-939211f6-126f-4150-8b7e-d2b3360effff.png)
+
+- Install vscode extensions inside the remote container once it is opened, install the following extensions:
+    * [C++ Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack)
+    * [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
```
Comment (csokun) [inline KINESIS.md:9]: Add these recommended extensions to `.vscode/extenions.json` 
```json
{
  "recommendations": [EXENDION_NAME(s)]
}
```

File: docker/Dockerfile.kinesis [hunk 2]
```diff
@@ -0,0 +1,48 @@
+ARG DISTRO=focal
+ARG CORE_BUILDER_IMAGE=abxit/kinesis-core-builder:latest
```
Comment (csokun) [inline docker/Dockerfile.kinesis:2]: TODO: kinesis-core-builder might need to bundle rust

File: docker/Dockerfile.kinesis [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+VOLUME /data
+VOLUME /postgresql-unix-sockets
+
+ADD docker/setup /setup.sh
+RUN ./setup.sh
+
+# We need to repeat ARG here to make it available inside build context
+# See https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
+ARG DISTRO=focal
+ARG STELLAR_CORE_VERSION=18.5.0-877.d387c6a71.focal
```
Comment (csokun) [inline docker/Dockerfile.kinesis:34]: version bump

File: install-rust.sh [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+# We download rustup-init from a URL adjacent to the SHA256 file above, and
+# check that it matches the expected SHA256 wired-in to this file, and then run
+# it.
+#
+# Rustup with then install Rust binary components (compiler, stdlib, etc.) and
+# check their PGP signatures match the Rust project's signing key (the signing
+# key is embedded in rustup).
+rm -f rustup-init
+curl --fail --output rustup-init "https://static.rust-lang.org/rustup/archive/${RUSTUP_VERSION}/${HOST_TRIPLE}/rustup-init"
```
Comment (csokun) [inline install-rust.sh:53]: The location is moved? check with @charu-gera

File: src/herder/Upgrades.cpp [hunk 5]
```diff
@@ -277,11 +304,16 @@ Upgrades::toString() const
...
+
     appendInfo("protocolversion", mParams.mProtocolVersion);
     appendInfo("basefee", mParams.mBaseFee);
     appendInfo("basereserve", mParams.mBaseReserve);
+    appendInfo("basepercentagefee", mParams.mBasePercentageFee);
+    appendInfo("maxfee", mParams.mMaxFee);
     appendInfo("maxtxsetsize", mParams.mMaxTxSetSize);
     appendInfo("flags", mParams.mFlags);
+    appendInfo("basepercentagefee", mParams.mBasePercentageFee);
+    appendInfo("maxfee", mParams.mMaxFee);
```
Comment (csokun) [inline src/herder/Upgrades.cpp:316]: duplicate

File: KINESIS.md (hunk 1 above)
Comment (charu-gera) [reply KINESIS.md:9]: fixed in https://github.com/bullioncapital/kinesis-core/pull/24/commits/5dc397aac8c33bea463a20252b401f7341e7c36c

File: docker/Dockerfile.kinesis (hunk 2 above)
Comment (charu-gera) [reply docker/Dockerfile.kinesis:2]: as discussed raised ticket https://bullioncapital.atlassian.net/browse/BLC-514 in backlog

File: docker/Dockerfile.kinesis (hunk 3 above)
Comment (charu-gera) [reply docker/Dockerfile.kinesis:34]: fixed in https://github.com/bullioncapital/kinesis-core/pull/24/commits/c9b8e4202d724246c742c2696e14a756b35616c7

**Pull request closed without merging**